src/AndroidManifest.xml

```diff
@@ -12,6 +12,17 @@
         android:allowBackup="true">
         <activity android:name="com.microsoft.aad.adal.AuthenticationActivity" >
         </activity>
+        
+        <receiver
```

Comment:
yup, i will add it in the readme when we're ready to ship the new broker. I also want to know the version of the two broker apps with the PRT feature and have a note on the readme in order not to confuse people.